.gitignore

```diff
@@ -34,4 +34,5 @@ yarn-error.log*
 # vercel
 .vercel
 
-# hello world test
\ No newline at end of file
+# hello world test
+.env
```

Comment:
I think we can remove these lines since `.env` is added further up on line 28! Also, looking at the [enviornment variables](https://nextjs.org/docs/basic-features/environment-variables) section of the docs I think we can switch to using `.env.local` vs `.env` as the file to hold our environment variables  👀